client/app/components/debugger/debugger.component.ts

```diff
@@ -50,6 +51,9 @@ export class DebuggerComponent {
             .map(fileContents => {
                 this.lines = fileContents.split('\n')
                     .filter((l, i) => this.sourceFunction && i >= (this.sourceFunction.lineStart - 1) && i < (this.sourceFunction.lineStart - 1 + this.sourceFunction.lineCount));
+                this.lines = this.lines.map(function(l) {
```

Comment:
It should b/c if I understand it correctly then when an expression spans multiple lines then each line will get the highlighting.